modules/era/lua/era_status_effects.lua

```diff
@@ -0,0 +1,20 @@
+-----------------------------------
+-- Common Requires
+-----------------------------------
+require('modules/module_utils')
+-----------------------------------
+-- Module definition
+-----------------------------------
+-- This module attempts to have a centralized file to override status effects with era changes.
+local eraStatusEffects = Module:new('era_status_effects')
+
+-----------------------------------
+-- Module enable/disable
+-----------------------------------
+eraStatusEffects:setEnabled(false)
```

Comment:
I'm probably going to get rid of `setEnabled`, if you load the module file - it'll be enabled. So this little section can be removed

Reply:
Maybe a comment in the header saying when it changed - if you can find that info!

Reply:
- Removed section
- Added date change
- Included link